flowpipe/graph.py

```diff
@@ -5,10 +5,10 @@
     from collections import OrderedDict
 except ImportError:
     from ordereddict import OrderedDict
+from concurrent import futures
```

Comment:
This still needs to be added to the requirements

Reply:
Sorry, ignore that, had overlooked your comment

Reply:
Let's add it to the setup.py as described in your link and see if the tests on travis run through